src/profile/v100/profile_uniffi_fn.rs

```diff
@@ -186,4 +197,32 @@ mod uniffi_tests {
             false
         );
     }
+
+    #[test]
+    fn check_if_encrypted_profile_json_contains_legacy_p2p_links_when_p2p_links_are_present(
+    ) {
+        let json =
+            serde_json::to_vec(&EncryptedProfileSnapshot::sample()).unwrap();
+        let password = "babylon";
+        assert_eq!(
```

Comment:
if you run `cargo clippy --fix --allow-staged` you will notice it will change this to `assert!(..)` only